kalite/facility/settings.py

```diff
@@ -1,5 +1,5 @@
 try:
-    import local_settings
+    from kalite import local_settings
```

Comment:
Hoping this idiom is a temporary solution, yes? Since we don't want every app depending on the kalite app?